packages/braintree-integration/src/BraintreeLocalPaymentMethod.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { FunctionComponent, useEffect } from 'react';
+
+import {
+    PaymentMethodProps,
+    PaymentMethodResolveId,
+    toResolvableComponent,
+} from '@bigcommerce/checkout/payment-integration-api';
+
+const BraintreeLocalPaymentMethod: FunctionComponent<any> = ({
+    method,
+    checkoutService,
+    paymentForm,
+    language,
+    onUnhandledError,
+}) => {
+    const methodId = method.id;
+
+    const initializePayment = async () => {
```

Comment:
Why did not you use `useCallback` here and the other places?

Reply:
Was made same as here https://github.com/bigcommerce/checkout-js/blob/4e78956966d5107437a4e6fbc1fe767fe0e9456c/packages/paypal-commerce-integration/src/components/PayPalCommercePaymentMethodComponent.tsx#L30

Reply:
Everything is fine. Discussed in private messages